plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt

```diff
@@ -265,6 +284,26 @@ open class KtlintPlugin : Plugin<Project> {
         project.tasks.findByName("check")?.dependsOn(ktlintCheck)
     }
 
+    private fun addApplyToIdeaTask(target: Project, extension: KtlintExtension) {
+        target.afterEvaluate {
+            val rootProject = target.rootProject
+            if (rootProject.tasks.findByName(APPLY_TO_IDEA_TASK_NAME) == null) {
+                val ktLintConfig = createConfiguration(rootProject, extension)
+                // -y here to auto-overwrite existing IDEA code style
+                val runArgs = mutableSetOf("--apply-to-idea-project", "-y")
+                addAndroidArg(extension, runArgs)
+
+                target.rootProject.taskHelper<JavaExec>("ktlintApplyToIDEA") {
```

Comment:
This is kinda inconsistent with the gradle `idea` task naming scheme.

Reply:
addressed